pkg/meta/redis_metrics.go

```diff
@@ -27,9 +27,15 @@ var (
 		Name: "transaction_restart",
 		Help: "The number of times a transaction is restarted.",
 	})
+	opDist = prometheus.NewHistogram(prometheus.HistogramOpts{
```

Comment:
Should we just the existing `txDist`?

Reply:
Then the existing grafana should just works.

Reply:
I think they are two different things: one is meta operation, the other one is engine write transaction, which is more likely to be the trouble maker. Also txRestart is a relational item with txDist, but not that close to opDist.

Reply:
Grafana should be compatible since txDist is still there with no modification

Reply:
We care more about on all the meta operations rather than some write operation, the transaction is kind of implementation details, that's why we prefer the operation latency rather than transaction details, same in Grafana.

Once we have opDist to cover all the operations, txDist is less useful, so I'd suggest to combine them together (the txDist is almost identical to  opDist for write operations). 

The txRestart is also useful to know the conflict between update opeartions.